### PR TITLE
Replace multiselect combobox addon with core component

### PR DESCRIPTION
**Pull request by paodb**

### CODE REVIEW

File: lookup-field-flow/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.vaadin.componentfactory</groupId>
     <artifactId>lookup-field-flow</artifactId>
-    <version>23.1.3</version>
+    <version>23.0.2</version>
```
Comment (mlopezFC) [inline lookup-field-flow/pom.xml:9]: I think it should be 23.2.0 and not 23.0.2, am I right?

**Review by mlopezFC — CHANGES_REQUESTED** [review]
Can you check my comment?

File: lookup-field-flow/pom.xml (hunk 1 above)
Comment (paodb) [reply lookup-field-flow/pom.xml:9]: Yes, you're right.

Comment (paodb) [reply lookup-field-flow/pom.xml]: Done

**Review by mlopezFC — APPROVED** [review]
LGTM

**Pull request merged**